examples/ola-recorder.cpp

```diff
@@ -117,20 +123,14 @@ int VerifyShow(const string &filename) {
   map<unsigned int, unsigned int> frames_by_universe;
   uint64_t total_time = 0;
 
-  unsigned int universe;
-  ola::DmxBuffer buffer;
-  unsigned int timeout;
+  ShowEntry entry;
   ShowLoader::State state;
   while (true) {
```

Comment:
I think it would probably good if verify also respected, and printed your in and out times. So you can find out how many frames per universe within your time range.